### ISSUE TITLE
add time estimation for pauses (so you know when to insert magnets / nuts ...)

### ISSUE THREAD

**Issue by discip** [SoftFever/OrcaSlicer#2191]
### Is your feature request related to a problem? Please describe.
To be able to put things like _bearings, magnets, nuts_ ... into the print, it is essential to know at what place in time that specific layer is reached to put the respective part in and resume the print.
Due to the lack of any time indication, it is currently a relatively tedious task to _'calculate'_ (it it is more likely **guessing**) when printing will be paused.

### Which printers will be beneficial to this feature
All of them! 😊

### Describe the solution you'd like
- [ ] 1. (green)
To address the initial issue we would need separate entries in the **_'Total estimation - Info Board'_** for each individual pause.
- [x] 2. (yellow)
Additionally, as proposed here #2045 there could be time estimation per layer as implemented in [PS](https://github.com/prusa3d/PrusaSlicer).
- [ ] 3. (red)
And to round things off, this request #679 could be taken into account to make the user's life even easier by letting him know when exactly the pause is going to take place instead of having to calculate that time.
_In short: Not the time between print-start and the pause, but the **exact local time**._
- [ ] 4. (purple)
And while we're at it: why not add the appropriate layer alongside?
<br>

I marked the parts mentioned above accordingly.
<br>
![Screenshot from 2023-09-30 12-48-47](https://github.com/SoftFever/OrcaSlicer/assets/53649486/3ea9190f-7d03-453c-a2b1-679f1c7ba7fc)

### Current procedure
1. Remove the upper part above the layer, where the respective part (e.g. nut) should be inserted by cutting it,
2. set **`Top shell layers/thickness`** to **`0`**,
3. slice &
4. calculate the time

Hope this makes sense! 😃

thanks in advance
kind regards

**Comment by discip** [comment]
@niklasb22
I noticed your PR wich addresses to some extent what I'm asking for here.
At least regarding the location of the information.

Do you think this is doable?
If so, may I dare to politely ask whether you could try to implement what is at issue here, as I am not nearly as skilled to be able achieve that?

thanks in advance
kind regards

**Comment by discip** [comment]
@Noisyfox
First of all:
I hope you don't mind me tagging you. If you do however, please let me know and I won't do this again.

Do you think this request is feasible?
If so, may I dare to politely ask whether you could try to implement what is at issue here, as I am not able to do this myself?

**By the way: I appreciate your dedication, which is very notable in your attention to detail.** 👍

thanks in advance
kind regards

**Comment by Noisyfox** [comment]
I've actually never used the pausing feature so far, but I can see the value of the feature you proposed (and those related issues). Will take a look.

PS: feel free to tag me as you want.

**Comment by discip** [comment]
## Thank you very much! Very kind of you! 😃

You should definitely try this! Especially in combination with neodymium magnets!

I am very excited already!
